graylog2-web-interface/src/components/users/UserCreate/EmailFormGroup.jsx

```diff
@@ -25,7 +25,7 @@ const EmailFormGroup = () => (
                    maxLength={254}
                    type="email"
                    required
-                   help="Give the contact email address." />
+                   help="Give the contact an email address." />
```

Comment:
I think this was meant in a different way: "an email address to contact the user" "contact email".

I guess we should use something like "The user's email address." (same for the Full Name)

Reply:
Ahh yes, good point. I've adjusted it. How does it look now in https://github.com/Graylog2/graylog2-server/pull/9520/commits/7a9cccf556d9e68f4ce06f7584d27104ed425fe0?